src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Formatting/CSharpFormattingPassBase.cs

```diff
@@ -307,6 +308,16 @@ protected static bool ShouldFormat(FormattingContext context, TextSpan mappingSp
 
             return true;
 
+            bool IsRazorComment()
+            {
+                if (owner.IsCommentSpanKind())
+                {
+                    return true;
+                }
+
+                return false;
```

Comment:
nit: Can this just be `bool IsRazorComment() => owner.IsCommentSpanKind()`?

Reply:
It could, but I don't like it when expression bodied local functions and normal functions are mixed together. My brain finds it harder to scan 🤷‍♂️